.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,98 @@
+should target the default branch. Our current default branch is the **develop** branch, which serves as our integration branch.
+
+Pull requests should:
 
-## Pull-request guidelines
-* target the **develop** branch for integration
 * ensure code builds successfully.
 * do not break existing test cases
 * new functionality will only be merged with new unit tests
-  * new unit tests should integrate within the existing googletest framework.
-  * tests must have good code coverage
-  * code must also have benchmark tests, and performance must approach the compute bound limit or memory bound limit.
+* new unit tests should integrate within the existing googletest framework.
+* tests must have good code coverage
+* code must also have benchmark tests, and performance must approach the compute bound limit or memory bound limit.
+
+### Deliverables ###
+
+For each new file, please include the licensing header
+```
+/*******************************************************************************
+ * Copyright (c) 20xx Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+```
+
+### Process ###
+
+hipBLAS uses the clang-format tool for formatting code in source files.
+
+To format a file, use:
+
+```
+clang-format -style=file -i <path-to-source-file>
+```
+
+To format all files, run the following script in hipBLAS directory:
+
+```
+#!/bin/bash
+git ls-files -z *.cc *.cpp *.h *.hpp *.cl *.h.in *.hpp.in *.cpp.in | xargs -0 clang-format -style=file -i
+```
+
+Also, githooks can be installed to format the code per-commit:
+
+./.githooks/install
+
+## Code License ##
+
+All code contributed to this project will be licensed under the license identified in the [LICENSE.md](../LICENSE.md). Your contribution will be accepted under the same license.
```

Comment:
This link also doesn't work for me, looks like rocBLAS uses full path.

Reply:
The link works for me.